pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -582,17 +576,18 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, IControlled
     // AUM management fees
 
     /**
-     * @notice Returns the management AUM fee percentage as an 18-decimal fixed point number.
+     * @notice Returns the management AUM fee percentage as an 18-decimal fixed point number and the timestamp of the
+     * last collection of AUM fees.
      */
-    function getManagementAumFeePercentage() public view returns (uint256) {
-        return _managementAumFeePercentage;
-    }
+    function getManagementAumFeeParams()
+        public
+        view
+        returns (uint256 aumFeePercentage, uint256 lastCollectionTimestamp)
+    {
+        (aumFeePercentage, lastCollectionTimestamp) = ManagedPoolAumStorageLib.getAumFeeFields(_aumState);
 
-    /**
-     * @notice Returns the timestamp of the last collection of AUM fees.
-     */
-    function getLastAumFeeCollectionTimestamp() external view returns (uint256) {
-        return _lastAumFeeCollectionTimestamp;
+        // If we're in recovery mode then we bypass any fee logic by setting the fee percentage to zero.
+        aumFeePercentage = ManagedPoolStorageLib.getRecoveryModeEnabled(_poolState) ? 0 : aumFeePercentage;
```

Comment:
```suggestion
        // If we're in recovery mode then we bypass any fee logic by setting the fee percentage to zero.
        if (ManagedPoolStorageLib.getRecoveryModeEnabled(_poolState)) {
            aumFeePercentage = 0;
        }
```

Reply:
Huh, I had this as it saved bytecode in #1890. Seems like the opposite here though.